fenix/app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -159,7 +159,7 @@
     <string name="recently_visited_menu_item_remove">Remove</string>
 
     <!-- Content description for the button which navigates the user to show all of their history. -->
-    <string name="past_explorations_show_all_content_description_2">Show all past explorations</string>
+    <string name="past_explorations_show_all_content_description_2">Show all past explorations button</string>
```

Comment:
Saw that we already have another similar [Show all recent tabs button](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/firefox-android/fenix/app/src/main/res/values/strings.xml#138) string but this is not the direction we want to go.
We could look into adding `Role.Button` to the currently used [ClickableText](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt#110-112) or we could maybe refactor the `ClickableText` to a `TextButton` with this ensuring that the expected role is set and read by a11y services.